src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import static core.basesyntax.FigureType.CIRCLE;
+import static core.basesyntax.FigureType.ISOSCELESTRAPEZOID;
+import static core.basesyntax.FigureType.RECTANGLE;
+import static core.basesyntax.FigureType.RIGHTTRIANGLE;
+import static core.basesyntax.FigureType.SQUARE;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    static final int RADIUS_CIRCLE = 10;
+    private static final String NAME_FIGURE = "circle";
+    private static final String WHITE_COLOR = Color.WHITE.name().toLowerCase();
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public FigureAbstract getDefaultFigure() {
+        return new Circle(NAME_FIGURE, WHITE_COLOR, RADIUS_CIRCLE);
+    }
+
+    public FigureAbstract getRandomFigure() {
+        FigureType[] figures = FigureType.values();
+        int index = random.nextInt(figures.length);
+        FigureType figureType = figures[index];
+        if (figureType == SQUARE) {
+            return new Square(figureType.name().toLowerCase(),
+                    colorSupplier.getRandomColor().toLowerCase(),
+                    (int) (Math.random() * 101));
```

Comment:
use nextInt instead of casting to int. Also, you may consider adding +1 because geometric property probably can't be zero, and randomizer can return zero. If logic seems too complex, you may extract it to a private utility method